docs/CLI.md

```diff
@@ -52,64 +54,59 @@ cov versions
 cov serve
 ```
 * Starts up the ng server with needed flags
-* Same as `ng serve --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true`
 
 ```
-cov serve-e2e
+cov serve-prod
 ```
 * Starts the ng server with flags needed to run e2e coverage
-* Same as `node --max_old_space_size=5048 ./node_modules/.bin/ng serve --aot --prod --sourcemap=true --build-optimizer --proxy-config proxy.conf.json`
+* Same as `ng serve --proxy-config proxy.conf.js --sourceMap=true --aot --prod --optimization`
 
 ```
 cov e2e
 ```
 * Runs the end to end tests against local code base
-* Same as `ng e2e --config --proxy-config e2e/proxy.conf.json`
+* Same as `ng e2e`
```

Comment:
don't we still need the proxy conf on here?

Reply:
I believe the ng cli looks at the angular.json and finds the proxy config relative to the `sourceRoot`, which is the `e2e` dir.